Quick-Start-Guide.md

```diff
@@ -0,0 +1,130 @@
+Here is a quick start guide to get you up and running with your RomM instance!  The guide will be split into 3 parts:
+
+- Prepare - preparing the environment before executing any code.
+- Build
+- Configure
+
+---
+
+### Prepare
+
+This guide will assume that you already have the following done, if not - stop here and come back when you do.
+
+1. Docker and Docker Compose installed on your system of choice.
+2. OpenSSL on the machine that will run RomM.
+3. A Twitch.tv account.
+4. 2 Factor Authentication set up on your Twitch account. *This is a requirement to get a dev account and an IGDB key*
+5. A MobyGames account. *(optional)
+6. Roms organized in the correct format.
+
+##### Getting an IGDB Key
+
+This will allow you to generate a key to pull the metadata for your roms using Twitch's IGDB.
+
+1. Navigate to [Twitch's dev site](https://dev.twitch.tv/https:/).
+2. Hit ****Log in with Twitch** in the top right of the screen and log in with your Twitch account.
+3. Select **Applications** on the left hand menu, followed by the ***Register Your Application*** button.
+
+![](assets\Register_Application.png)
+
+4. You will now be brought to the registration screen, fill out the following information.
+
+   1. Name: A **Globally Unique** name for your application.  If you try to enter a name that already exists, it won't work and gives you an unhelpful error message.  I recommend following this format: Romm-(MD5 Hash of current date in UTC), as it is nearly impossible that it exists already. e.g. *RoMM-f3e5102b2d4e87cd283fc65907e8a8f0*.
+   2. OAuth Redirect URLs: The FQDN where your RomM instance will be hosted.  You only need to specify http://localhost if you intend to run/access it locally, otherwise you should enter the FQDN of your domain, this needs to be accesible via https.
+   3. Category: Application Integration.
+   4. Client Type: Confidential.
+
+![](assets\IGDB_Creation.png)
+
+5. Click ***Create*** to create the application.
+6. You will now be taken back to the **Applications** screen, select ***Manage*** next to the application you just created.
+7. Copy the ****Client ID**** near the bottom of the page.
+8. Select **New Secret** to generate a client secret for us to use.
+9. Copy both of these somewhere for safe keeping, we will need them later.  Obviously do not share these with anyone, the demo values here were removed after success.
+
+![](assets\IGDB_Secret.png)
+
+##### Getting a MobyGames Key
+
+This will allow you to generate a key to pull the metadata for your roms using MobyGames. *Not Required*
+
+> To Be Implemented
+
+##### Generating the Secret Key
+
+This step will generate a key that is used in the authorization of RomM.  Without this, you will be unable to log in and use the platform.  This step **must** be done on the machine that will be hosting RomM.
```

Comment:
```suggestion
This step will generate a key that is used in the by RomM to gain authorized access to the MobyGames API.  Without this, you will be unable to log in and use the platform.  This step **must** be done on the machine that will be hosting RomM.
```

Or something to that affect.

Reply:
I am unclear about this comment - the ROMM_SECRET_KEY is for logging into ROMM, it has no relevance on MObyGames integration.